azure-pipelines.yml

```diff
@@ -0,0 +1,74 @@
+jobs:
+- job: Linux 
+  strategy:
+    matrix:
+      Node.js 0.10:
+        NODE_VERSION: '0.10'
+      Node.js 0.12:
+        NODE_VERSION: '0.12'
+      Node.js 1.8:
+        NODE_VERSION: '1.8'
+      Node.js 2.5:
+        NODE_VERSION: '2.5'
+      Node.js 3.3:
+        NODE_VERSION: '3.3'
+      Node.js 4.9:
+        NODE_VERSION: '4.9'
+      Node.js 5.12:
+        NODE_VERSION: '5.12'
+      Node.js 6.14:
+        NODE_VERSION: '6.14'
+      Node.js 7.10:
+        NODE_VERSION: '7.10'
+      Node.js 8.12:
+        NODE_VERSION: '8.12'
+      Node.js 9:
+        NODE_VERSION: '9'
+      Node.js 10:
+        NODE_VERSION: '10'
```

Comment:
Current config has a failure on this version not considered an overall failure. Is this sey up this way / is there a method to set that up? It is used a lot along with the nightlys as sometimes a bad version can get published which will stay until at least the next day.

Reply:
So yes, I could make that work.  What I would do is create a separate matrix for the ones that can fail and the build still be considered successful.  If one of those builds then fails, I can mark the build as Partially Successful.